manual/core/address_resolution/README.md

```diff
@@ -0,0 +1,99 @@
+## Address resolution
+
+Each node in the Cassandra cluster is uniquely identified by an IP address that the driver will use
+to establish connections.
+
+* for contact points, these are provided as part of configuring the `Cluster` object;
+* for other nodes, addresses will be discovered dynamically, either by inspecting `system.peers` on
+  already connected nodes, or via push notifications received on the control connection when new
+  nodes are discovered by gossip.
+
+
+### Cassandra-side configuration
```

Comment:
This section is very useful.  It's a common problem new users run into (not understanding why can't connect w/ driver but cqlsh works locally).